### PR TITLE
Rearrange image to handle extreme aspect ratio

### LINKED ISSUE #143: Could not recognize text. How to fix this?
Could not recognize text. How to fix this?
5041b267d90e5d187e445ebe4683d6aa

### LINKED ISSUE #145: Tall images not translated at all
For tall images text not translated at all

I have tested it on 10+ images

### PR DESCRIPTION
**Pull request by dmMaze**
Fix #145, fix #143

Rearrange image to square batches before feeding into network if it satisfies following conditions:
1. Extreme aspect ratio 
AND
2. Is too tall or wide for detect size

which are common to webtoon and vertical comics(条漫)

Test image0 https://user-images.githubusercontent.com/20344446/209716520-d4d4484c-b7e3-49e5-b8c0-effa7e214f5d.jpeg
Test image1 https://user-images.githubusercontent.com/49092250/209435351-186af25b-e3db-421f-9014-d5406bb41c90.png

For example, image1 will be rearranged like this:
rearrange
then feed into DBNet. 

Network output will be un-rearranged (logits in overlapped area will be averaged) before generating text polygons. 
DBNet is segmentation-based, presumably truncation caused by the rearrangement won't deteriorate the final text polygons generation if the text line map have all text lines detected even if they're truncated.

### CTD
vertical_comic0_ctd_bboxes
vertical_comic1_bboxes_ctd

### Default
I can't get image0 to work. Maybe default detection model can't recognize its fonts.

bboxes

### CODE REVIEW

**Comment by dmMaze** [comment]
Besides it should be able to handle arbitrary tall (or wide) input image

**Comment by My12123** [comment]
The text is too small. How to fix it?
